_timeline/2018-04-10-peer-to-peer-internet-workshops.md

```diff
@@ -0,0 +1,5 @@
+---
+title: Peer-to-Peer Internet Workshops
+date: 2018-04-10
+---
+First day of a six-module [workshop series at the Toronto Public Library](https://tomesh.net/peer-to-peer-internet/), where participants experiment with peer-to-peer applications over a physical mesh network following our [open-source syllabus](https://github.com/tomeshnet/p2p-internet-workshop) designed and faciliated by Toronto Mesh volunteers.
```

Comment:
Timeline entries past tense! Also, consider using the full name of workshop in the title?

something like:

`First day of a six-module [workshop at the Toronto Public Library](https://tomesh.net/peer-to-peer-internet/), where participants experimented with peer-to-peer applications over a physical mesh network. Workshop used our [open-source syllabus](https://github.com/tomeshnet/p2p-internet-workshop) created by Toronto Mesh volunteers.`